test/busted/outputHandlers/nvim.lua

```diff
@@ -193,6 +194,9 @@ return function(options)
     io.write(globalTeardown)
     io.write(suiteEndString:format(testCount, tests, fileCount, files, elapsedTime_ms))
     io.write(getSummaryString())
+    if failureCount > 0 or errorCount > 0 then
+      io.write(global_helpers.read_nvim_log(nil, true))
+    end
```

Comment:
isn't it better to have this in either `handler.testEnd()` or `handler.error()`?

otherwise, I don't see much use of dumping the tail of the log. however, it would work if `read_nvim_log()` was grepping for `test_id`

Reply:
1. We've tried testEnd() (that's the current state of `master`). It is too noisy. This PR intentionally avoids that.
2. The tail should be short because most tests don't log (CI sets loglevel=WARN)
3. The logs in theory may be useful because of the test ids

> if `read_nvim_log()` was grepping for `test_id`

That could be interesting. At this point the idea is just to see if it's useful at all, and then improve things if **and only if** need becomes clear.